onnxruntime/python/tools/transformers/convert_beam_search.py

```diff
@@ -939,6 +942,115 @@ def convert_model(args: argparse.Namespace):
     logger.info(f"model save to {args.output}")
 
 
+def convert_greedy_search_model(args: argparse.Namespace):
+    """Convert model according to command line arguments.
+
+    Args:
+        args (argparse.Namespace): arguments parsed from command line
+    """
+    is_gpt2: bool = args.model_type == "gpt2"
+    if is_gpt2:
```

Comment:
The export onnx part is same as beam search. They can move to a function and shared by both.